src/js/contentUtils.ts

```diff
@@ -247,7 +252,7 @@ export function storeFoundJS(scriptNodeMaybe: HTMLScriptElement): void {
 
   if (scriptNodeMaybe.getAttribute('type') === 'application/json') {
     try {
-      JSON.parse(scriptNodeMaybe.textContent!);
+      scriptNodeMaybe.textContent && JSON.parse(scriptNodeMaybe.textContent);
```

Comment:
This is the same case as the manifest parsing above, right? Is this difference intentional?

Reply:
Addressed